conductr_cli/sandbox_run.py

```diff
@@ -27,23 +26,25 @@ def run(args):
     """`sandbox run` command"""
     is_conductr_v1 = major_version(args.image_version) == 1
     features = sandbox_features.collect_features(args.features, args.image_version)
+    sandbox = sandbox_run_docker if is_conductr_v1 else sandbox_run_jvm
 
-    if is_conductr_v1:
-        run_result = sandbox_run_docker.run(args, features)
-    else:
-        run_result = sandbox_run_jvm.run(args, features)
+    run_result = sandbox.run(args, features)
 
     is_started, wait_timeout = wait_for_start(args, run_result)
     if is_started:
         if not is_conductr_v1:
-            start_proxy(run_result.nr_of_proxy_instances)
+            feature_ports = []
+            for feature in features:
+                feature_ports.extend(feature.ports)
+
+            proxy_ports = sorted([] + args.ports + feature_ports)
```

Comment:
The empty list `[]` is now removed.

I thought in Python `+` is similar to list `extend`, but that's not the case:

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ python3
Python 3.5.2 (default, Jun 29 2016, 13:43:58)
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> x = [1]
>>> x
[1]
>>> x + [2] + [3]
[1, 2, 3]
>>> x
[1]
```